packages/mgt-components/src/components/mgt-file-list/mgt-file-upload/mgt-file-upload.ts

```diff
@@ -118,69 +157,78 @@ export interface MgtFileUploadItem {
 export interface MgtFileUploadConfig {
   /**
    * MS Graph APIs connector
+   *
    * @type {IGraph}
    */
   graph: IGraph;
 
   /**
-   *  allows developer to provide site id for a file
-   *  @type {string}
+   * allows developer to provide site id for a file
+   *
+   * @type {string}
    */
   siteId?: string;
 
   /**
    * DriveId to upload Files
-   *  @type {string}
+   *
+   * @type {string}
    */
   driveId?: string;
 
   /**
    * GroupId to upload Files
-   *  @type {string}
+   *
+   * @type {string}
    */
   groupId?: string;
 
   /**
    * allows developer to provide item id for a file
-   *  @type {string}
+   *
+   * @type {string}
    */
   itemId?: string;
 
   /**
-   *  allows developer to provide item path for a file
-   *  @type {string}
+   * allows developer to provide item path for a file
+   *
+   * @type {string}
    */
   itemPath?: string;
 
   /**
    * allows developer to provide user id for a file
-   *  @type {string}
+   *
+   * @type {string}
    */
   userId?: string;
 
   /**
    * A number value indication for file size upload (KB)
-   *  @type {Number}
+   *
+   * @type {Number}
    */
   maxFileSize?: number;
 
   /**
-   *  A number value to indicate the number of files to upload.
-   *  @type {Number}
+   * A number value to indicate the number of files to upload.
+   *
+   * @type {Number}
    */
-  maxUploadFile?: Number;
+  maxUploadFile?: number;
 
   /**
    * A Array of file extensions to be excluded from file upload.
    *
-   *  @type {string[]}
+   * @type {string[]}
    */
   excludedFileExtensions?: string[];
 }
 
-// tslint:disable-next-line: completed-docs
+// eslint-disable-next-line @typescript-eslint/tslint/config
 interface FileWithPath extends File {
```

Comment:
There should be a comment added here that this is a hack to trick typescript into allowing us to write to the fullPath property